### ISSUE TITLE
"No column provided" error when using any autocomplete fields

### ISSUE THREAD

**Issue by psy-q** [cosmocode/dokuwiki-plugin-struct#305]
I've tried to upgrade our test instance to version 2017-06-13 (b730c13) but I'm having trouble. Any autocomplete fields will throw this error:

```
No column provided ajax.php:67
#0 /home/wiki/farm/farmer/lib/plugins/struct/action/ajax.php(42): action_plugin_struct_ajax->executeTypeAjax()
#1 /home/wiki/farm/farmer/inc/events.php(211): action_plugin_struct_ajax->handle_ajax(Object(Doku_Event), NULL)
#2 /home/wiki/farm/farmer/inc/events.php(70): Doku_Event_Handler->process_event(Object(Doku_Event), 'BEFORE')
#3 /home/wiki/farm/farmer/lib/exe/ajax.php(30): Doku_Event->advise_before()
#4 {main}
```

And the styling looks off as well:
![image](https://user-images.githubusercontent.com/87557/27167592-dcff4084-51a1-11e7-8400-86e9b0b8fb01.png)

I have saved the config so I think any cache should be cleared, and I used shift-reload in Firefox to prevent local caching.

**Comment by splitbrain** [comment]
@micgro42 I guess the weird styling stems from the recent label change. Looks like we forgot about the info thingy. Not sure where the Ajax error comes from.

**Comment by psy-q** [comment]
Wow, that was a quick fix 😮   Thanks a lot, I tested it and it works. I hope this doesn't undo some of the cleanups you had done.
